### PR TITLE
Update SANTIAGO-2025

### PR DESCRIPTION
**Pull request by kokedevops**
- Se agrega lugar del evento 
- Se agrega dia del evento
- Se agregan nuevos integrantes comite organizador
- Se agrega tabla de Sponsor

### CODE REVIEW

File: static/events/2025-santiago/organizers/caiomedeiros.png
Comment (toshywoshy) [inline static/events/2025-santiago/organizers/caiomedeiros.png:1]: Please move to the new location `assets/event/2025-santiago/organizers/`

File: static/events/2025-santiago/organizers/franciscomeneses.jpg
Comment (toshywoshy) [inline static/events/2025-santiago/organizers/franciscomeneses.jpg:1]: Please move to the new location `assets/event/2025-santiago/organizers/`

File: static/events/2025-santiago/organizers/jorgevalenzuela.png
Comment (toshywoshy) [inline static/events/2025-santiago/organizers/jorgevalenzuela.png:1]: Please move to the new location `assets/event/2025-santiago/organizers/`

File: static/events/2025-santiago/organizers/juanjosemendez.jpg
Comment (toshywoshy) [inline static/events/2025-santiago/organizers/juanjosemendez.jpg:1]: Please move to the new location `assets/event/2025-santiago/organizers/`

File: static/events/2025-santiago/organizers/karenquijada.png
Comment (toshywoshy) [inline static/events/2025-santiago/organizers/karenquijada.png:1]: Please move to the new location `assets/event/2025-santiago/organizers/`

File: static/events/2025-santiago/organizers/sergiocanales.png
Comment (toshywoshy) [inline static/events/2025-santiago/organizers/sergiocanales.png:1]: Please move to the new location `assets/event/2025-santiago/organizers/`

File: data/events/2025/santiago/main.yml [hunk 1]
```diff
@@ -58,7 +60,12 @@ team_members: # Name is the only required field for team members.
     image: "jorgevalenzuela.png"
   - name: "Caio Medeiros"
     image: "caiomedeiros.png"
-
+  - name: "Sergio  Canales"
+    image: "sergiocanales.png"  
+  - name: "Juan Jose Mendez"
+    image: "juanjosemendez.jpg"
+  - name: "Francisco Meneses"
+    image: "franciscomeneses.jpg"
```
Comment (toshywoshy) [inline data/events/2025/santiago/main.yml:69]: You are adding 3 new organizers, please email info@ and get their approval and make sure these people are also on the mailing list

**Review by toshywoshy — DISMISSED** [review]

File: data/events/2025/santiago/main.yml (hunk 1 above)
Comment (kokedevops) [reply data/events/2025/santiago/main.yml:69]: Fue enviado el correo a info y también se informo Dan

File: static/events/2025-santiago/organizers/caiomedeiros.png
Comment (kokedevops) [reply static/events/2025-santiago/organizers/caiomedeiros.png:1]: Move to new directory

File: static/events/2025-santiago/organizers/jorgevalenzuela.png
Comment (kokedevops) [reply static/events/2025-santiago/organizers/jorgevalenzuela.png:1]: Move to new directory

File: static/events/2025-santiago/organizers/franciscomeneses.jpg
Comment (kokedevops) [reply static/events/2025-santiago/organizers/franciscomeneses.jpg:1]: Move to new directory

File: static/events/2025-santiago/organizers/juanjosemendez.jpg
Comment (kokedevops) [reply static/events/2025-santiago/organizers/juanjosemendez.jpg:1]: Move to new directory

File: static/events/2025-santiago/organizers/sergiocanales.png
Comment (kokedevops) [reply static/events/2025-santiago/organizers/sergiocanales.png:1]: Move to new directory

File: static/events/2025-santiago/organizers/karenquijada.png
Comment (kokedevops) [reply static/events/2025-santiago/organizers/karenquijada.png:1]: Move to new directory

**Comment by nimbinatus** [comment]
The mailing list and Slack changes are complete.

File: content/events/2025-santiago/sponsor.md [hunk 2]
```diff
@@ -1,66 +1,196 @@
 +++
-Title = "Sponsor"
+Title = "Patrocinadores"
 Type = "event"
-Description = "Sponsor Devopsdays Santiago 2025"
+Description = "Patrocinadores DevOpsDays Santiago 2025"
 +++
 
-Valoramos enormemente los patrocinadores para este evento abierto. Si está interesado en patrocinar, envíenos un correo electrónico a [{{< email_organizers >}}].
+Para patrocinar la conferencia, envíanos un correo electrónico a [santiago@devopsdays.org](mailto:santiago@devopsdays.org).
```
Comment (nimbinatus) [inline content/events/2025-santiago/sponsor.md:7]: A non-blocking note for the future: The `[{{email_organizers}}]` code block adds the email address on build. No need to hardcode it in :)

**Pull request merged**